Assets/MRTK/Core/Interfaces/Services/IMixedRealitySystem.cs

```diff
@@ -28,6 +28,27 @@ public interface IMixedRealityService : IDisposable
         /// </remarks>
         BaseMixedRealityProfile ConfigurationProfile { get; }
 
+        #region State properties
+
+        /// <summary>
+        /// Indicates whether or not the service has been initialized.
+        /// </summary>
+        bool IsInitialized { get; }
+
+        /// <summary>
+        /// Indicates whether or not the service is currently enabled.
+        /// </summary>
+        bool IsEnabled { get; }
+
+        /// <summary>
+        /// Indicates whether or not the Destroy method been called on this service.
+        /// </summary>
+        bool IsMarkedDestroyed { get; }
+
+        #endregion State properties
```

Comment:
Since we're reviewing this, I wanted to start a conversation around the purpose of our interfaces.

For example, it seems reasonable for us to enforce "hey, a system needs to be able to be `Initialize`d" and have something useful and implementation-specific happen.

It seems almost boilerplate, should be handled by a base class or tracked by some manager, to make all systems also report their `IsInitialized` state. Can we depend on that bool being set by arbitrary inheritors? _Should_ we depend on that bool being set?